protocol-units/bridge/service/src/main.rs

```diff
@@ -42,10 +42,13 @@ async fn main() -> Result<()> {
 
 	tracing::info!("Bridge config loaded: {bridge_config:?}");
 
-	let one_stream = EthMonitoring::build(&bridge_config.eth).await.unwrap();
+	let (eth_health_tx, eth_health_rx) = tokio::sync::mpsc::channel(10);
+	let one_stream = EthMonitoring::build(&bridge_config.eth, eth_health_rx).await.unwrap();
```

Comment:
It's probably me but wouldn't `stream_a` or `stream_one` be a little easier to read

Reply:
I really don't know. We start with one and two chain, so if there isn't a good reason to change, we should stay like that.

Reply:
It's a language convention, regarding the ordering of words. `stream_one` is more English and `one_stream` is more French.

It's similar to how in English today's date is "November 6" but in French it's "Le 6 Novembre". 

I do tend to think because this code is based on English, having the number after the name of what is being numbered (eg `stream_one` and `stream_two`), could be helpful for consistency with English language conventions. I can understand it either way, but it's something worth considering.

Reply:
Ok I thought it was to replace `one` with `a` and `two` with `b` not the ordering.
Ok yes you're right it's better, I'll change it.